Source/Resources/Content/MSGraph/Groups/BUILTIN - Microsoft 365 Users - Insiders.json

```diff
@@ -2,7 +2,7 @@
   "assignedLicenses": {
     "$ref": "ResourceContext:M365Licenses"
   },
-  "description": "This manual group is used to define users in the insiders update ring.",
+  "description": "What is it: A manual group of users that are in the insiders ring. \n Simeon Notes: This group can be used to apply and test user directed polices.",
```

Comment:
**What does this do?**
Creates a manual "Insiders" group to which configurations can be deployed before other rings of users. The baselines creates the "Insiders" user group that can receive deployments before others users.

**Why should you use this?**
If you want to test policies in release rings to a subset of users (Insiders) before deploying to all users. 

**What is the end user impact?**
Users of this group can receive configuration changes before others users. 

**Learn more**
NA